src/Pages/ForgotPassword.cshtml.cs

```diff
@@ -52,7 +55,9 @@ public async Task<IActionResult> OnGet()
                 return Page();
             }
             Email = Request.Cookies["Email"];
-            var recoveryCode = (await EmailSender.SendRecoveryCodeAsync(Email)).ToString();
+
+            Configuration.GetSection(Utilities.EmailSender.Email).Bind(EmailSender.Value);
```

Comment:
Can you explain what this line does?

Reply:
So I'm trying out options pattern a bit more, and what this does is takes a section from app settings (GetSection(Utilities.EmailSender.Email)) where the section name is "Email" (from the const string in above question) and then binds it to the properties of the EmailSender object.
Basically this gets data from app settings and injects it into a object